lib/rule-context.js

```diff
@@ -92,18 +92,18 @@ function mergeFixes(fixes, sourceCode) {
     const start = fixes[0].range[0];
     const end = fixes[fixes.length - 1].range[1];
     let text = "";
-    let lastPos = Math.max(0, start);
+    let lastPos = Number.MIN_SAFE_INTEGER;
```

Comment:
Maybe it would be better to use `-Infinity` in case someone uses extremely small numbers in fix ranges for whatever reason, but I think it's probably fine -- files usually don't have 2<sup>53</sup> BOMs. 😄